src/Nordea/Components/Range.elm

```diff
@@ -0,0 +1,153 @@
+module Nordea.Components.Range exposing
+    ( Config
+    , Range
+    , init
+    , view
+    , withShowInterval
+    )
+
+-- CONFIG
+
+import Css
+    exposing
+        ( Style
+        , backgroundColor
+        , borderRadius
+        , cursor
+        , displayFlex
+        , flexDirection
+        , height
+        , justifyContent
+        , margin2
+        , marginTop
+        , pct
+        , pointer
+        , property
+        , pseudoElement
+        , rem
+        , row
+        , spaceBetween
+        , transparent
+        , width
+        )
+import Html.Styled as Html exposing (Attribute, Html, div, input, span)
+import Html.Styled.Attributes exposing (css, name, type_)
+import Html.Styled.Events exposing (onInput)
+import Nordea.Html exposing (showIf)
+import Nordea.Resources.Colors as Colors
+import Nordea.Themes as Themes
+
+
+type alias Config msg =
+    { value : Float
+    , min : Float
+    , max : Float
+    , step : Maybe Float
+    , onInput : String -> msg
+    , showInterval : Bool
+    }
+
+
+type Range msg
+    = Range (Config msg)
+
+
+init : Float -> Float -> Float -> (String -> msg) -> Range msg
+init value min max onInput =
+    Range
+        { value = value
+        , min = min
+        , max = max
+        , step = Nothing
+        , onInput = onInput
+        , showInterval = False
+        }
+
+
+withShowInterval : Bool -> Range msg -> Range msg
+withShowInterval value (Range config) =
+    Range { config | showInterval = value }
+
+
+
+-- VIEW
+
+
+view : List (Attribute msg) -> Range msg -> Html msg
+view attributes (Range config) =
+    div attributes
+        [ input
+            [ name "rangeInput"
```

Comment:
Do we need the name? I see it is used with the label's `for` attribute in Slider, but I think the `for` attribute should be used with id's instead of names. And id's should generally be avoided in a component because it means that the component can only be used once per page, which may not be expected. I suggest to wrap the input in the label instead


```suggestion
            [
```

Reply:
I want to remove these `name` and `for` attributes aswell 🤔 but how can i go on about it when the use in Slider is structured like this?

```
div[][
    div[][
        label[][]
        , numberInput[][]
    ]
    , Rangeinput [][]
]
```

Reply:
Cant find a way to wrap the label around the rangeInput with this setup

Reply:
This is the most semantically correct setup I can think of for this. I think the label for the two inputs may be visually hidden, but needs to be there for accessibility reasons

```elm
fieldset []
    [ legend [] [ Html.text "The label for the whole component" ]
    , label [] [ Html.text "Label for the numberinput", numberInput [] [] ]
    , label [] [ Html.text "Label for the rangeinput", rangeInput [][] ]
]
```